### DISCUSSION TITLE
Consider renaming packages to follow PascalCase convention

### DISCUSSION THREAD

**Discussion by glen-84** [xunit/xunit#3070] [Ideas]
The xunit (and coverlet) packages are the only packages with lower-case names in my solution. All other packages use PascalCase.

Would you consider renaming the packages to `XUnit.*`, maybe for v3?

I know that it's not a huge issue, but consistency is important.

**Comment by bradwilson** [comment]
Consistency with whom? Our packages have been lowercase since NuGet first started, and our filenames have been lowercase longer than that.

**Reply by glen-84** [reply]
Consistency with the vast majority of other packages on NuGet, including ~95% of the top 100.

**Reply by bradwilson** [reply]
I understood your point. I just disagree with it.
